content/docs/rfcs/51/README.md

```diff
@@ -66,11 +66,8 @@ shard clusters are divided into ranges:
 
 | index (range) |    usage                 |
 |    ---        |    ---                   |
-|             0 |   global                 |
-|        1 - 15 |   reserved               |
-|     16 - 1023 |   specific app protocols |
-|  1024 - 49152 |   all app protocols      |
-| 49152 - 65535 |   automatic sharding     |
+|        0 - 15 |   reserved               |
```

Comment:
Should we reserve `1` here for the public Waku Network?

Reply:
indeed, added here: fcf719c3e7e077f99cfd156d742faf249ec3bb25
(in rfc 52)